### DISCUSSION TITLE
Minecraft closes when i change dimension

### DISCUSSION THREAD

**Discussion by ricardonunezvillacis** [maruohon/litematica#609] [Q&A] [answered]
Hi, i've a problem with Litematica, my minecraft closes when i change dimension
In reported error of Java shows me this

#
# A fatal error has been detected by the Java Runtime Environment:
#
#  EXCEPTION_ACCESS_VIOLATION (0xc0000005) at pc=0x00007ffc8e550c60, pid=1548, tid=1732
#
# JRE version: OpenJDK Runtime Environment Microsoft-32931 (17.0.3+7) (build 17.0.3+7-LTS)
# Java VM: OpenJDK 64-Bit Server VM Microsoft-32931 (17.0.3+7-LTS, mixed mode, tiered, compressed oops, compressed class ptrs, g1 gc, windows-amd64)
# Problematic frame:
# C  [nvoglv64.dll+0x1850c60]
#
# No core dump will be written. Minidumps are not enabled by default on client versions of Windows
#
# If you would like to submit a bug report, please visit:
#   https://github.com/microsoft/openjdk/issues
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

**How can fix it? Please help me, I like it this mod**

**Comment by maruohon** [comment, marked as answer]
See here: https://github.com/maruohon/litematica/issues/601

TL;DR; The new nvidia driver update broke stuff, use the previous version for now.
